www/blog/2022-11-23-library-ts-perf.md

```diff
@@ -0,0 +1,197 @@
+---
+slug: library-ts-perf
+# TODO: better title (pls help me)
+title: How can you improve the performance of your TypeScript library?
```

Comment:
You could write this for a wider audience than only people who write libs

![download](https://user-images.githubusercontent.com/459267/203752200-e5852125-1ebe-4b24-b78f-dd32fd00ab62.png)